src/app/_components/client/UserListItem/UserListItem.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+
+import { useToggle } from "@/app/_hooks";
+import { UserType } from "@/app/_types/response/user";
+
+import Avatar from "../Avatar/Avatar";
+import ConfirmModal from "../ConfirmModal/ConfirmModal";
+import { useToggleFollow } from "./hooks";
+
+import { twJoin } from "tailwind-merge";
+
+interface UserListItemProps {
+  type: "following" | "follow";
+  user: UserType;
+  isFollowing: boolean;
+  isPrivate: boolean;
+}
+
+const UserListItem = ({ type, user, isFollowing, isPrivate }: UserListItemProps) => {
+  const { isToggle, handleToggleOff, handleToggleOn } = useToggle();
```

Comment:
```suggestion
  const { 
isToggle : isShowModal, handleToggleOff : closeModal, handleToggleOn : openModal
} = useToggle();
```

useToggle이 공용 훅이다 보니까 정확히 어떤걸 토글 하는건지 알지 못하는 점에서 코드를 읽기 아려웠던것 같아서 위처럼 변경해줘도 좋을것같아요!

Reply:
네이밍 명시적으로 수정해 놓겠습니다 !! 👍